org.hedgewars.Hedgewars.yaml

```diff
@@ -0,0 +1,399 @@
+app-id: org.hedgewars.Hedgewars
```

Comment:
Please reformat the file according to our yaml styleguide: https://github.com/flathub/flathub/wiki/YAML-Style-Guide

Reply:
Done